azavea/azavea-branding-guide
# Adds GroundWork and new DistrictBuilder logos

## Overview
GroundWork is a new product, and we needed to add it and its identity to the guide for easy access. The DistrictBuilder brand has recently undergone a refresh, so this PR updates existing content.

### Demo
<img width="1792" alt="Screen Shot 2020-08-06 at 4 21 53 PM" src="https://user-images.githubusercontent.com/5672295/89550002-fb8ad200-d800-11ea-84e2-b4b768e85ca0.png">
<img width="1792" alt="Screen Shot 2020-08-06 at 4 21 58 PM" src="https://user-images.githubusercontent.com/5672295/89550007-fcbbff00-d800-11ea-8949-b28c2304c0d3.png">

### Notes
- I removed the "in progress" tags on certain pages since they've been present for a long time & the pages are probably done. 

## Testing Instructions
- `git pull`
- Go to the most recent release note at the bottom of the sidebar
- Click the links to make sure they work, at the same time view the GroundWork and DistrictBuilder sections in the Products section.
- Click the Download buttons for each on the Products section. Ensure that the name of the zip looks right.
- Navigate to the Logo Downloads page.
- Click the Download buttons for each here. Ensure that the name of the zip looks right.

Connects https://github.com/azavea/azavea-marketing/issues/112, Closes #122
